src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -44,31 +44,11 @@ const shouldPullDrupal = buildOptions =>
   !fs.existsSync(getDrupalCachePath(buildOptions));
 
 function pipeDrupalPagesIntoMetalsmith(contentData, files) {
-  const {
-    data: {
-      nodeQuery: { entities: pages },
-    },
-  } = contentData;
-
-  const skippedContent = {
-    nullEntities: 0,
-    emptyEntities: 0,
-  };
+  const pages = contentData.data.nodeQuery.entities.filter(
```

Comment:
Honestly, I don't know if this has to do with unpublished entities or not. After fixing up the transformers, `yarn build:content --use-cms-exports` made it past that stage and into the rest of `getDrupalContent`, but was running into issues with `addHubIconField` failing because an item in `pages` was null. It's unclear to me why this wasn't failing before, as I'm pretty sure `pages` contained null items previously, but this change ensures that all entries in `pages` are neither null nor 
empty objects `{}`. We were skipping them anyhow, so I figured it was safe to filter them out.